### ISSUE TITLE
[Bug Report][3.8.7] V-Data-Table with 'show-select' needs aria label for form controls

### ISSUE THREAD

**Issue by kmmerc18** [vuetifyjs/vuetify#21526]
### Environment
**Vuetify Version:** 3.8.7
**Vue Version:** 3.5.13
**Browsers:** Chrome 137.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Make a data table with the show-select prop to show the checkbox column. Run the axe devtools over this example

### Expected Behavior
Every form control element (i.e. the checkbox inputs) should have a unique aria-label so the axe tool should return no issues

### Actual Behavior
There are no aria-labels on the checkbox <input> elements within the v-data-table

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNrNlkGL00AUx7/KYy6r0KSpdbGWurAuIiLKwq4s2PbwmjybcSeTMJm0SslZBFdQz+LJo9/Nj+Ck2W7ThM6Wetn00M77v5f34/U/kwwX7DhJ3FlGrM8GmqJEoKajkQQYzJwANToaJ4KKAMDMieKAxJMRS0mQrykYsVLpc1ObGmH5vYoWC2eGIiOjSIxoJaRhPHfKexSBo0G72sy0H7QrLGaZ+ooneslFH5JYaQjoHWZCw6K8Y1EN9+6vlgCKdKbkem2aXjP3YThurcNLYhNbR6BaVVwFeh8O/v66+gpmXIIOKvXFJWIfNY+lSbrANORyqmNZTwqJT0NtUjy3U9cmmBYdPNd7VJdmsciW7Y3oebXKfGO1FfsLPEVpPhbuZ36GQaws0A+2Qx/aoWuVO0J/gucKE0ot0C80io8W5HrnKnND2mTec9Kf4cKYVkUkmgaogJ8YjzT+jQr4w63cXRt2TdyR+QpOuSS8xdjnIXKBMrBQN9huqO3DrtlnR+zvcBKSUtzqkPNMXdKdssjvP/AK5TS22VoG3OaOTmO/3TA3TpZN5P0m/QPOtML5xEy7McoK9puzY9ugt7ujKdUGvd8B8g1OCf3wP7ah93g7c8/OXNvBO3rjJ7zkc24hfk1zeEt4y0ZsHsg7n9UWT6+fmnn5oxTNYtC+fkCzFhNcXqbu+zSW5pViMWJ+at4H+sNxbjQeFc9uJ8KkklAGi6RFnrO8xbruodvpmvSu23N7rKVVRuN/4TQsEA==)

I know there was a ticket about this in the past, but it was for an earlier Vuetify and Vue version, and this issue is still occurring in our production environment

**Comment by noahfizz** [comment]
@mueller-jens Has this been merged into the library? I saw your MR for a fix, but was not able to understand if it was merged or not. If it was, how do I implement the ability to add aria-labels to the show-select checkboxes per row on a v-data-table?

**Comment by mueller-jens** [comment]
@noahfizz As you can see at the merge request wasn't merged and I do not think it never will be.

**Comment by noahfizz** [comment]
@mueller-jens Well that is heartbreaking. Do you happen to know of any workaround?
